pkg/bloombuild/builder/metrics.go

```diff
@@ -22,5 +29,25 @@ func NewMetrics(r prometheus.Registerer) *Metrics {
 			Name:      "running",
 			Help:      "Value will be 1 if the bloom builder is currently running on this instance",
 		}),
+
+		taskStarted: promauto.With(r).NewCounter(prometheus.CounterOpts{
+			Namespace: metricsNamespace,
+			Subsystem: metricsSubsystem,
+			Name:      "task_started_total",
+			Help:      "Total number of task started",
+		}),
+		taskCompleted: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Namespace: metricsNamespace,
+			Subsystem: metricsSubsystem,
+			Name:      "task_completed_total",
+			Help:      "Total number of task completed",
+		}, []string{"status"}),
+		taskTime: promauto.With(r).NewHistogramVec(prometheus.HistogramOpts{
+			Namespace: metricsNamespace,
+			Subsystem: metricsSubsystem,
+			Name:      "task_time_seconds",
+			Help:      "Time spent processing a task.",
+			Buckets:   prometheus.DefBuckets,
```

Comment:
The default bucket range might be too small.

Reply:
Indeed. Changed